README.rst

```diff
@@ -66,3 +66,4 @@ Reporting Issues
 To report an issue with Salt, please follow the guidelines for filing bug reports:
 
 * http://docs.saltstack.com/en/latest/topics/development/reporting_bugs.html
+ping
```

Comment:
You can trigger a Jenkins build by posting "Go Go Jenkins!" as a comment. See @jtand 's comment.